### DISCUSSION TITLE
Issue while running npm test

### DISCUSSION THREAD

**Discussion by MatteoFilosa** [uwdata/mosaic#221] [General]

**Comment by MatteoFilosa** [comment]
I cloned the github repo for mosaic, I install the dependencies with `npm i` and then run `npm test`. However, I get this error everytime: 
![mosaic issue](https://github.com/uwdata/mosaic/assets/67047355/5ec4e3c4-143b-44aa-935d-0708e2dfb345)

How can I solve this issue? Thanks in advance.

**Reply by jheer** [reply]
Hmm, for some reason it looks like the arrow extension installation is not happening on your system.

One thing to try: before running tests, try launching the duckdb server from the command line `npm run server`, which should then perform any initial installations. You can then kill the server via `Ctrl-C` and try running the tests again?

p.s. it looks like you are running on Windows? Not sure if that is related to the issue, but we develop on MacOS and test (via CI on GitHub) on Linux. So if there is a Windows specific issue we may be unaware of it on our end.

**Comment by MatteoFilosa** [comment]
I tried running `npm run server`, but I got: 
![image](https://github.com/uwdata/mosaic/assets/67047355/f3d6dbf8-3d35-4efc-a8f2-8ed2d96691dc)
And yes, I am running on Windows.

**Reply by domoritz** [reply]
This means the arrow extension doesn't have prebuilt packages for your windows platform. Looks like it may not be there for newer duckdb versions either: https://github.com/duckdb/arrow/issues/20.

**Comment by domoritz** [comment]
This is an older discussion and possibly resolved with updates in DuckDB. I'm going to close this but can reopen if it's still an issue.
